android/build.gradle

```diff
@@ -39,16 +39,6 @@ android {
       }
     }
   }
-
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_11
-    targetCompatibility JavaVersion.VERSION_11
-  }
-
-  kotlinOptions {
-    freeCompilerArgs = ['-Xjvm-default=all']
```

Comment:
Thanks @yungblud - is there any reason you also want to remove `kotlinOptions.freeCompilerArgs`, or would it be sufficient for your app to remove just the strict Java dependencies to allow for Java 17?

Reply:
Hmm, let me test it.
I think if just removing compileOptions is good, that is the best way.

Reply:
Yeah, it's okay to remain freeCompilerArgs.
That was my bad.

But weirdly, I pushed onto this branch but github didn't apply it to PR.

Reply:
I will close this PR, and reopen it.
I think this is github's bug.

Reply:
https://github.com/braze-inc/braze-react-native-sdk/pull/242